### PR TITLE
Add usage example to README.md

### PR DESCRIPTION
**Pull request by kutyel**
This might seem trivial but in my experience is critical for adoption! 

Thanks for this awesome library, I'll never format cabal files by hand again! 🚀

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,17 @@
...
+```sh
+$ cabal install cabal-fmt
+$ cabal-fmt example.cabal > example-formatted.cabal
 ```
+
+## Output
+
+Turns this...
+
+```yaml
```
Comment (phadej) [inline README.md]: it's not `yaml`, GitHub knows how to highlight `cabal` format.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,17 @@
-Turns
+# cabal-fmt
 
+## Usage
+
+```sh
+$ cabal install cabal-fmt
+$ cabal-fmt example.cabal > example-formatted.cabal
```
Comment (phadej) [inline README.md]:
```
% cabal-fmt --help
cabal-fmt - .cabal file reformatter

Usage: cabal-fmt [-i|--inplace] 
                 [--Werror | --Wno-error | --indent N | --tabular | 
                   --no-tabular] [FILE...] [--version]
  Reformat .cabal files

Available options:
  -i,--inplace             process files in-place  <-- !!! 
  --Werror                 Treat warnings as errors
  --indent N               Indentation
  --tabular                Tabular formatting
  FILE...                  input files
  -h,--help                Show this help text
  --version                Show version
```

File: README.md [hunk 3]
```diff
@@ -4,14 +4,30 @@
 
 ```sh
 $ cabal install cabal-fmt
-$ cabal-fmt example.cabal > example-formatted.cabal
+$ ...
```
Comment (phadej) [inline README.md]: I meant rather that there's `--inplace` flag (which the arrow was pointing at).

File: README.md [hunk 4]
```diff
@@ -5,22 +5,7 @@
...
-
-Available options:
-  -i,--inplace             process files in-place  <-- !!! 
-  --Werror                 Treat warnings as errors
-  --indent N               Indentation
-  --tabular                Tabular formatting
-  FILE...                  input files
-  -h,--help                Show this help text
-  --version                Show version
+$ cabal-fmt -i example.cabal
```
Comment (phadej) [inline README.md]: :+1:

Comment (kutyel) [reply README.md]: Ready for merge! 😄

**Comment by phadej** [comment]
Squashed in https://github.com/phadej/cabal-fmt/commit/1741edc0671c7428d6a92a1501b1ac09b0c404dd, thanks

**Pull request closed without merging**